packages/zapp/console/src/components/Launch/LaunchForm/LaunchFormDialog.tsx

```diff
@@ -0,0 +1,48 @@
+import { Dialog } from '@material-ui/core';
+import * as React from 'react';
+import { LaunchForm } from 'components/Launch/LaunchForm/LaunchForm';
+import { ResourceIdentifier, ResourceType } from 'models/Common/types';
+import {
+  TaskInitialLaunchParameters,
+  WorkflowInitialLaunchParameters,
+} from 'components/Launch/LaunchForm/types';
+
+function getLaunchProps(id: ResourceIdentifier) {
+  if (id.resourceType === ResourceType.TASK) {
+    return { taskId: id };
+  } else if (id.resourceType === ResourceType.WORKFLOW) {
+    return { workflowId: id };
+  }
+  throw new Error('Unknown Resource Type');
+}
+
+export const LaunchFormDialog: React.FC<{
+  className?: string;
```

Comment:
same here, calssName is never used even if provided. We shouldn't add props just in case, until we know that the would be used.

Reply:
fixed